clang/include/clang/Basic/Features.def

```diff
@@ -104,6 +104,7 @@ FEATURE(thread_sanitizer, LangOpts.Sanitize.has(SanitizerKind::Thread))
 FEATURE(dataflow_sanitizer, LangOpts.Sanitize.has(SanitizerKind::DataFlow))
 FEATURE(scudo, LangOpts.Sanitize.hasOneOf(SanitizerKind::Scudo))
 FEATURE(ptrauth_intrinsics, LangOpts.PointerAuthIntrinsics)
+FEATURE(ptrauth_qualifier, LangOpts.PointerAuthIntrinsics)
```

Comment:
This looks strange?

Reply:
Why is it strange? Because it uses `LangOpts.PointerAuthIntrinsics` as the predicate?

Reply:
Oh, right, nevermind.  They both use LangOpts.PointerAuthIntrinsics because they're both tied to the ptrauth support.

Reply:
Yes, that's right. We currently don't have a separate flag or predicate for ptrauth qualifier.